test/rest_request_template/travis_correct_string_json.json

```diff
@@ -6,6 +6,7 @@
     "snowflake.url.name":"SNOWFLAKE_HOST",
     "snowflake.user.name":"SNOWFLAKE_USER",
     "snowflake.private.key":"SNOWFLAKE_PRIVATE_KEY",
+    "snowflake.private.key.passphrase":"${file:./apache_properties/file-secrets.txt:PASSPHRASE}",
```

Comment:
This test demos that reading from file works. If snowflake.private.key.passphrase is not replaced with empty string in the file, then the connector will throw error of not being able to parse the private key with the passphrase.